src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,15 +17,22 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
```

Comment:
@wjwithagen Yes, it is helpful for diagnostic purposes.  I would prefer that we didn't remove it.